hawk/app/lib/hb_report.rb

```diff
@@ -38,6 +38,21 @@ def running?
     Util.child_active(@pidfile)
   end
 
+  def report_generated?
+    return { code: false, msg: '' } if !File.exist?(@exitfile) || !File.exist?(@timefile)
+
+    rc = File.read(@exitfile).to_i
+    return { code: true, msg: '' } if rc==0
+
+    lt = lasttime
+    errfile = Rails.root.join('tmp', 'reports', "hawk-#{lt[0]}-#{lt[1]}.stderr").to_s
+    # .stderr file must exist even if there was no error
+    return { code: false, msg: '' } if !File.exist?(errfile)
+    
+    err_msg = File.new(errfile).read.split("\n")[0]
```

Comment:
Can you use the last line instead? User-friendly messages usually appear at the last line of the stderr, since crmsh uses exceptions to handle errors.

Reply:
Hi, I changed the code. The prompt will show both the first and the last line.